geoutils/raster/raster.py

```diff
@@ -2189,7 +2189,6 @@ def save(
 
         :returns: None.
         """
-        dtype = self.data.dtype if dtype is None else dtype
```

Comment:
Why did you need to delete this line?

Reply:
Because `dtype` was never used later in the function (`mypy` strangely didn't pick that up but some of my other IDE linters did)

Reply:
Haha ok... In that case, it should be removed from the argument list. Or we could use the argument to force the output type to a preferred type. I will raise an issue.